cpp/src/parquet/types.h

```diff
@@ -479,6 +479,9 @@ struct Encoding {
   };
 };
 
+// Exposed data encodings.
+enum class ExposedEncodingType { NO_ENCODING = 0, DICTIONARY = 1 };
```

Comment:
please add a comment for no_encoding indicating converts all encoded data to a flat list.